R/badge.R

```diff
@@ -78,6 +78,32 @@ stages <- c(
   questioning = "blue"
 )
 
+#' Create a Binder badge
+#'
+#' This prints out the markdown which will display a Binder "badge", indicating
+#' launch build your repository source codes on Binder <https://mybinder.org/>.
```

Comment:
This part of the sentence still doesn't make sense to me:

> indicating launch build your repository source codes on Binder

I visited <https://mybinder.org> and tried to understand what's going on. Does this sound correct?

> indicating that your repository can be launched in an executable environment on <https://mybinder.org/>.

If that's OK, let's use that wording (or something similar).

Reply:
Thank you for your suggestion. That is the tidy phrase :)